tests/test_cache_utils.py

```diff
@@ -558,6 +547,7 @@ def test_push_to_hub_overwrite_existing(self):
             # With a directory
             with self.assertLogs("optimum", level="INFO") as cm:
                 push_to_cache_on_hub(neuron_hash, cache_dir, self.CUSTOM_PRIVATE_CACHE_REPO)
+                print(cm.output)
```

Comment:
In similar tests you removed the print instruction. Do you still need it here ? Maybe it is because in other places you have inserted a mark_step ?

Reply:
I think I leave this mostly for debugging purposes when I want to see the actual log output (which was captured by the context-manager if I recall properly). 

What you are referring to is different: basically before we were using `print` to force graph compilation. A print being equivalent to a `xm.mark_step()` + a computation of the resulting graph + transfert of the output to the CPU. Since the last compiler update it seems to not be working in this specific case, so just replaced it with `xm.mark_steps()`.